packages/components/src/components/toggle-switch/toggle-switch.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes                      from 'prop-types';
+import classNames                     from 'classnames';
+
+class ToggleSwitch extends Component {
+    render() {
+        return (
+            <Fragment>
+                <input
+                    className={classNames('toggle-switch', {
+                    }, this.props.className)}
+                    id={'dt_toggle_switch'}
```

Comment:
How can we change this to prevent use of `id` in shared components? As `id` should only exist once in a page, we wouldn't want it to be duplicated, right?